src/cljs/main/broadfcui/common/icons.cljs

```diff
@@ -89,3 +95,12 @@
    :render
    (fn [{:keys [props]}]
      (icon {:style {:color (:color props) :fontSize (:size props)}} :warning))})
+
+(react/defc UnknownIcon
```

Comment:
Yeah, but I wanted to match the other icons. Should I just refactor all of them? It'd be fairly trivial.